src/content/popup/expandable.ts

```diff
@@ -64,6 +64,9 @@ export function updateExpandable(
       `calc(${round(collapsedHeight, 2)}px + var(--expand-button-allowance))`
     : `${expandedHeight}px`;
 
+  // TODO: temporary solution in development bc this isn't playing well with React
+  expandable.style.height = '600px';
```

Comment:
For some reason, the `.expandable` element was collapsing to `height: 0px` whenever I switched the kanji entry to React. I'm not sure why yet, but I'm guessing something to do with the React element not yet being rendered when the expandable does its height measurement. I'll dig into it more tomorrow so I can get rid of this.

Reply:
Yes, this part is tricky (it took quite a bit of work to get the expandable behavior to behave correctly in all the different configurations) so I'd definitely prefer to deal with this in a separate PR where we can take the time to get it right.